textattack/constraints/grammaticality/language_models/google_language_model/google_language_model.py

```diff
@@ -96,7 +100,7 @@ def get_probs(x, x_adv_list):
         # same order they were passed in.
         max_el_indices.sort()
         
-        return np.array(x_adv_list)[max_el_indices]
+        return list(np.array(x_adv_list)[max_el_indices])
```

Comment:
Why does this have to be a list?

If it really does have to be a list, then it's not necessarily faster to use the masking of nparrays in this case, and maybe we should just do
```
return [x_adv_list[i] for i in max_el_indices]
```

Maybe the masking is nicer though, and the performance difference should be negligible. up to you.

Reply:
It's a list to stay consistent with the call_many method it overrides. Good point, I'll change it to the list comprehension way.